### PR TITLE
Scenario docs

### LINKED ISSUE #1755: Update scenario docs with eval6 baseline results

### PR DESCRIPTION
**Pull request by davidslater**
Fixes #1755

@swsuggs here is where I left off with scenario documentation. Let's chat tomorrow about this.

### CODE REVIEW

**Comment by swsuggs** [comment]
Thoughts on this?

I believe it's essentially complete besides filling in some numbers in a few results files.

I decided to leave poison.md as its own separate document because it contains much detail that seemed like it would be out of place in scenarios.md.  I still need to add a description of Sleeper Agent.

**Comment by davidslater** [comment]
Can you point me to the readthedocs link for it?

File: docs/scenarios.md [hunk 1]
```diff
@@ -1,579 +1,466 @@
...
-500-600 are in the validation split, and 600-700 are in the test split.
+  * Armory includes one dataset suited for ASR:
+    * [LibriSpeech dataset](http://www.openslr.org/12) (custom subset)
 * **Baseline Model:**
-To maximize time spent on defense research, a trained baseline model will be provided, but
-performers are not required to use it, if their defense requires a different architecture.
-The model is an ImageNet-pretrained DenseNet-121 that is fine-tuned on RESISC-45.
+Armory includes an audio model using the [DeepSpeech 2](https://arxiv.org/pdf/1512.02595v1.pdf) architecture with
+pretrained weights from either the AN4, LibriSpeech, or TEDLIUM datasets.  Custom weights
+may also be loaded by the model.
```
Comment (davidslater) [inline docs/scenarios.md:39]: There is also a HuBERT model available - see the eval6 scenario configs. I would add a deprecation warning to Deep Speech 2, and say that we'll be removing that in armory 0.17.

File: docs/baseline_results/librispeech_asr_results.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+# Audio ASR Baseline Evaluation:
```
Comment (davidslater) [inline docs/baseline_results/librispeech_asr_results.md:1]: Add a "Coming Soon" section for the HuBERT baseline (see eval6 scenario configs)

**Review by davidslater — COMMENTED** [review]
Looks good to me. Two minor comments below. Let's add those, merge into rc0.16.1, and then test that one on readthedocs.

Since I made the PR, you'll need to approve to merge.

File: docs/scenarios.md (hunk 1 above)
Comment (swsuggs) [reply docs/scenarios.md]: done

File: docs/baseline_results/librispeech_asr_results.md (hunk 2 above)
Comment (swsuggs) [reply docs/baseline_results/librispeech_asr_results.md:1]: done

**Comment by swsuggs** [comment]
One carla MOT run did not complete.  Besides that, the poisoning ones need to be filled in (most of the results already exist except for re-running cifar stuff with the new model), and it looks like librispeech audio classification never had baseline results before.  We can merge this and add the rest of the results as resolution to issue #1755.

**Pull request merged**